### PR TITLE
table creation

### PR DESCRIPTION
**Pull request by ramico**
stats
elegant result

### CODE REVIEW

File: includes/class-wc-siftscience-admin.php [hunk 1]
```diff
@@ -163,9 +163,26 @@ private function output_settings_stats() {
...
+			$stats = json_decode( $stats , true );
+			$stats_array = $stats;
 			$stats = json_encode( $stats, JSON_PRETTY_PRINT );
-			echo '<pre>' . $stats . '</pre>';
+
+			$stats_keys = array_keys( $stats_array );
+						
+			for ( $i = 0 ; $i < count( $stats_keys ) ; $i++ ) { 
+				$key = $stats_keys[ $i ];
+				$present_key = str_replace( '_', ' ', $key );
```
Comment (nabsul) [inline includes/class-wc-siftscience-admin.php:174]: this string is actually the name of the function. So I don't think it makes sense to make it look like "English".

Also, I was thinking of changing the format of the string to look like: `[class name]::[function name]`

File: includes/class-wc-siftscience-admin.php [hunk 2]
```diff
@@ -163,9 +163,26 @@ private function output_settings_stats() {
...
+				
+				echo "<table><thead>
+						<tr><th colspan=\"2\" style=\"text-align:left\">$present_key:</th></tr>
```
Comment (nabsul) [inline includes/class-wc-siftscience-admin.php:177]: How did you decide on the indentation of this line and the next one?

File: includes/class-wc-siftscience-admin.php [hunk 3]
```diff
@@ -163,9 +163,26 @@ private function output_settings_stats() {
...
 
-			$stats = json_decode( $stats );
+			$stats = json_decode( $stats , true );
+			$stats_array = $stats;
 			$stats = json_encode( $stats, JSON_PRETTY_PRINT );
-			echo '<pre>' . $stats . '</pre>';
+
+			$stats_keys = array_keys( $stats_array );
+						
+			for ( $i = 0 ; $i < count( $stats_keys ) ; $i++ ) {
```
Comment (nabsul) [inline includes/class-wc-siftscience-admin.php:172]: This for-loop can be simplified with a foreach-loop

File: includes/class-wc-siftscience-admin.php [hunk 4]
```diff
@@ -158,14 +158,25 @@ private function output_settings_stats() {
...
 				echo '<p>No stats stored yet</p>';
 				return;
 			}
 
-			$stats = json_decode( $stats );
-			$stats = json_encode( $stats, JSON_PRETTY_PRINT );
-			echo '<pre>' . $stats . '</pre>';
+			$stats = json_decode( $stats , true );
+
+			foreach ( array_keys( $stats ) as $outer_v ) {
```
Comment (nabsul) [inline includes/class-wc-siftscience-admin.php:168]: Even better: `foreach ( $stats as $key => $value )`.

File: includes/class-wc-siftscience-admin.php [hunk 5]
```diff
@@ -158,14 +158,25 @@ private function output_settings_stats() {
...
+										
+				echo '<table><thead>
+					<tr><th colspan="2" style="text-align:left">' . $outer_v . ':</th></tr>
+					</thead><tbody>';
+
+				foreach ( array_reverse( $stats[ $outer_v ] ) as $inner_k => $inner_v ) {
```
Comment (nabsul) [inline includes/class-wc-siftscience-admin.php:174]: Why array_reverse?

Comment (ramico) [reply includes/class-wc-siftscience-admin.php:174]: time is four chars long unlike count looks better 
can be changed

File: includes/class-wc-siftscience-admin.php (hunk 4 above)
Comment (ramico) [reply includes/class-wc-siftscience-admin.php:168]: I'm iterating keys i have no need for value (array structure )

Comment (nabsul) [reply includes/class-wc-siftscience-admin.php:168]: Your are using the value a little late

File: includes/class-wc-siftscience-admin.php (hunk 5 above)
Comment (nabsul) [reply includes/class-wc-siftscience-admin.php:174]: Ok, it's fine.

Comment (nabsul) [inline includes/class-wc-siftscience-admin.php]: You use the value here

File: includes/class-wc-siftscience-admin.php (hunk 4 above)
Comment (ramico) [reply includes/class-wc-siftscience-admin.php:168]: ???

**Pull request merged**